src/components/pages/docs.yml

```diff
@@ -31,6 +28,12 @@ user:
 actions:
   en: Actions
   de: Aktionen
+clone:
+  en: Clone
+  de: Klonen
+delete:
```

Comment:
You might find those general button labels also already in `common.yml` (not "clone" though)

Reply:
True, reused translation for deletion, thank you